Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -4,6 +4,9 @@
 using SEE.Controls;
 using SEE.Utils;
 using SEE.GO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace SEE.UI
```

Comment:
The menu entry `Exit` must be renamed to `Exit Game` and a confirmation dialog must be added if this menu entry is selected: exiting the game is a non-reversible action and the user must be clear about that.